src/lang/gettext_string_hash.hpp

```diff
@@ -0,0 +1,69 @@
+//
+// Created by machaviprace on 22.07.21.
+//
+#pragma once
+
+#include <stdio.h>
+#include <stdint.h>
+
+#define HASHWORDBITS 32
+
+//template <uint32_t TEST>
+class CTracker {
+public:
+    CTracker(volatile int32_t *duration);
```

Comment:
`volatile` keyword is deprecated in newer C++ standards, do you REALLY need the `volatile` keyword here to avoid ALL optimizations accessing the parameter `duration`?

Reply:
Yes, I wanted to avoid all optimalization, beacuse I wanted to be sure that I can read the parameter from debuger